tests/all/traps.rs

```diff
@@ -539,6 +547,7 @@ fn parse_dwarf_info() -> Result<()> {
 
 #[test]
 #[cfg_attr(all(target_os = "macos", target_arch = "aarch64"), ignore)] // TODO #2808 system libunwind is broken on aarch64
+#[cfg_attr(target_os = "openbsd", ignore)] // backtraces are broken on OpenBSD
```

Comment:
Could this perhaps be extracted to a helper function in this module that tests for arm64 macos and openbsd and skips the tests if that's true?